Porting/release_managers_guide.pod

```diff
@@ -578,7 +577,10 @@ blead release, so you may find nothing to do here.
 
 =head3 Update AUTHORS
 
-The AUTHORS file can be updated by running F<Porting/updateAUTHORS.pl>.
+The AUTHORS file can be updated by running:
+
+ $ perl Porting/updateAUTHORS.pl
```

Comment:
Personal preference, but I think that 4 spaces stands out more for verbatim test. It's also consistent with Markdown syntax.

Reply:
Almost all other verbatim are standing out 1 space.